kit/score/registry_example_test.go

```diff
@@ -23,13 +23,13 @@ func fibonacci(n uint) uint {
 var scoreRegistry = score.NewRegistry()
 
 func init() {
-	scoreRegistry.Add(TestFibonacciMax, len(fibonacciTests), 20)
-	scoreRegistry.Add(TestFibonacciMin, len(fibonacciTests), 20)
+	scoreRegistry.Add(TestFibonacciMax, "", len(fibonacciTests), 20)
```

Comment:
Shouldn't these be updated to use `AddWithTaskName()` and `AddSubWithTaskName()`

Reply:
Or rather the `""` should be removed to be compatible with the previous API, which has been reverted.

Reply:
Seems I forgot to revert these when I updated the API again.
Will be changed back in next push.